features/step_definitions/scaffold.rb

```diff
@@ -7,3 +7,7 @@
 Then(/content of a file named (.+) in a directory (.+) should be equal to (.+)/) do |file, dir, content|
     assert_equal File.read(File.join(@dir, dir, file)), content
 end
+
+Then(/content of a file named (.+) in a directory (.+) should be equal to:$/) do |file, dir, content|
```

Comment:
mm, I might be wrong, but I don't see a capturing group for `content` in the sentence. Should it be?

```rb
/content of a file named (.+) in a directory (.+) should be equal to: (.+)$/
```

Reply:
The syntax seems to be different for multiline strings: http://asymmetrical-view.com/2011/06/02/cucumber-gherkin-and-multiline-arguments.html